threshsign/src/ThresholdSignaturesTypes.cpp

```diff
@@ -229,8 +226,7 @@ void Cryptosystem::validatePrivateKey(const std::string& key) const {
   isValidKey(keyType, key, expectedKeyLength);
 }
 
-bool Cryptosystem::isValidCryptosystemSelection(const std::string& type, const std::string& subtype) {
-  UNUSED(subtype);
+bool Cryptosystem::isValidCryptosystemSelection(const std::string& type, [[maybe_unused]] const std::string& subtype) {
```

Comment:
Since we are not currently using [[maybe_unused]] in the code, either keep the UNUSED macro or refactor each usage of it.
Since these are places which we would potentially like to refactor (because their interface is not fully used), it will be easier to do so by searching a single term.
This comment applies to all of the other places you used this attribute.